lib/nth_prime_cases.rb

```diff
@@ -0,0 +1,19 @@
+class NthPrimeCase < OpenStruct
+  def name
+    'test_%s' % description.downcase.gsub(/[ -]/, '_')
+  end
+
+  def do
```

Comment:
I really don't like that there is a method called `do`, 
I see it a lot, is there documentation that uses an example with `do` in it? 

I'd prefer it be called `actual` to fit the `assert_equal expected, actual` pattern but am open to other suggestions for better names.